From 9b8132738c364fc3c886e81e7d383aaff80dc867 Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Sat, 5 Dec 2015 12:00:53 +0100
Subject: [PATCH] Disable tests

Tests are causing build failures on some architectures that are missing
a proper fenv.h, so just disable them.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

---
Status: Buildroot specific, not suitable for upstream in this state.
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 5137c9e..eae79fd 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,4 +1,4 @@
-SUBDIRS = pixman demos test
+SUBDIRS = pixman demos
 
 pkgconfigdir=$(libdir)/pkgconfig
 pkgconfig_DATA=pixman-1.pc
-- 
1.9.1

